src/bootstrap/build/dist.rs

```diff
@@ -61,9 +61,11 @@ pub fn docs(build: &Build, stage: u32, host: &str) {
 
     // As part of this step, *also* copy the docs directory to a directory which
     // buildbot typically uploads.
-    let dst = distdir(build).join("doc").join(&build.package_vers);
-    t!(fs::create_dir_all(&dst));
-    cp_r(&src, &dst);
+    if host == build.config.build {
+        let dst = distdir(build).join("doc").join(&build.package_vers);
+        t!(fs::create_dir_all(&dst));
+        cp_r(&src, &dst);
+    }
```

Comment:
Yeah this is `build/doc`, which I think is just what we upload to the S3 bucket, which in turn buildbot filters to just upload one of these? I was hoping the `host == build.config.build` would filter this build at least and we'd only copy one into place.